_pages/en/docs.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+ref: Docs
+lang: en
+permalink: /docs.html
+---
+
+<h1 property="name" id="wb-cont">Open Resource Exchange Data</h1>
+
+<h3 class="h5">Directory structure</h3>
+
+<ul>
+    <li>Administrations: List of Canadian federal, provincial, territorial and municipal administrations. Used to lookup bilingual names.</li>
+    <li>code: List of released open source code by administrations. Sub-folders contain data files. One file per administration.</li>
+    <ul>
+        <li>federal</li>
+        <li>provincial</li>    
+        <li>municipal</li>    
+    </ul>
+    <li>open_source_software: List of open source software used by administrations.</li>
+</ul>
+
+<h3 class="h5">How to add your data</h3>
```

Comment:
I think all of the content in the [README](https://github.com/canada-ca/ore-ero/blob/master/_data/README.md) can be reused for

- Adding your administration's information
- Adding Open Source Code
  - Your administration is already there
  - Your administration needs to be added
- Adding Open Source Software
  - The software is already there
  - Your open source software needs to be added
- Adding Open Standards...

Add links to schemas under code examples

Reply:
is it necessary to add the code examples in docs.html page? because there is a link to a code example in every metadata schema page